QEfficient/generation/text_generation_inference.py

```diff
@@ -18,6 +19,27 @@
 from QEfficient.utils import padding_check_and_fix
 from QEfficient.utils.logging_utils import logger
 
+
+@dataclass
+class CloudAI100ExecInfo:
+    """
+    holds all the information about Cloud AI 100 execution
+    :param generated_texts: List[str]
+    :generated_ids: np.ndarray
+    :prefill_time: float
+    :decode_perf: float
+    :total_perf: float
+    :total_time: float
+    """
+
+    generated_texts: List[str]
+    generated_ids: np.ndarray
+    prefill_time: float
+    decode_perf: float
+    total_perf: float
+    total_time: float
+
```

Comment:
add `__repr__` method, which can be used for printing the stats of execution just by printing or logging the object of this class.